test/man/checkManPages

```diff
@@ -218,4 +235,4 @@ def _error(*args):
 
 
 if __name__ == '__main__':
-    main()
+    main()
```

Comment:
I don't know whether Python is sensitive to terminating newlines or not, but I'd probably restore this.